src/datasources/cache/cache.service.interface.ts

```diff
@@ -14,4 +14,6 @@ export interface ICacheService {
   deleteByKey(key: string): Promise<number>;
 
   deleteByKeyPattern(pattern: string): Promise<void>;
+
+  expire(key: string, expireTimeSeconds: number);
```

Comment:
Since we now have `expire` as a separate function what do you think about removing that functionality from `set` in order to make these operations more granular? (for another PR)

Reply:
Yes, I think it makes perfect sense, we can do that in another PR, thanks! 👍